src/model/index.ts

```diff
@@ -66,6 +75,16 @@ class Model implements WorkbookModel {
     if ((value?.object_type ?? "").startsWith("TX_STOCK_")) {
       this.TX_STOCK(value);
     }
+
+    if ((value?.object_type ?? "").startsWith("TX_PLAN_SECURITY_")) {
+      this.TX_PLAN_SECURITY(value);
+    }
+  }
+
+  public consumePendingTransactions() {
+    for (const object of this.pendingTransactionsBySecurityId_) {
+      this.consume(object);
+    }
```

Comment:
**Question:** Are we still using this or can we delete?

Reply:
We can delete it. Will do that in the next PR